internal/elfexec/elfexec.go

```diff
@@ -208,13 +208,13 @@ func GetBase(fh *elf.FileHeader, loadSegment *elf.ProgHeader, stextOffset *uint6
 		if loadSegment.Vaddr == start-offset {
 			return offset, nil
 		}
-		if start >= loadSegment.Vaddr && limit > start && (offset == 0 || offset == pageOffsetPpc64) {
+		if start >= loadSegment.Vaddr && limit > start && (offset == 0 || offset == pageOffsetPpc64 || offset == start) {
 			// Some kernels look like:
 			//       VADDR=0xffffffff80200000
 			// stextOffset=0xffffffff80200198
 			//       Start=0xffffffff83200000
 			//       Limit=0xffffffff84200000
-			//      Offset=0 (0xc000000000000000 for PowerPC64)
+			//      Offset=0 (0xc000000000000000 for PowerPC64) (==Start for Linux Perf)
```

Comment:
I think this case corresponds to when KASLR is active, so I'd reflect that in the comment and PR description. Some experimentation:
```
$ sudo cat /proc/kallsyms | grep _stext
ffffffff866002b8 T _stext
$ sudo perf record sleep 5
[ perf record: Woken up 1 times to write data ]
[ perf record: Captured and wrote 0.018 MB perf.data (7 samples) ]
$ pprof -raw perf.data | tail -3
Mappings
1: 0x0/0x1/0x0 perf_4.9 <build ID redacted> 
2: 0xffffffff86600000/0xffffffffc030a000/0xffffffff86600000 [kernel.kallsyms] <build ID redacted>
```
Then rebooted the machine and did the same (note changed addresses):
```
$ sudo cat /proc/kallsyms | grep _stext
[sudo] password for aalexand: 
ffffffff892002b8 T _stext
$ sudo perf record sleep 5
[ perf record: Woken up 1 times to write data ]
[ perf record: Captured and wrote 0.018 MB perf.data (2 samples) ]
$ pprof -raw perf.data | tail -3
Mappings
1: 0x0/0x1/0x0 perf_4.9 <build ID redacted> 
2: 0xffffffff89200000/0xffffffffc0338000/0xffffffff89200000 [kernel.kallsyms] <build ID redacted>
```